packages/connect/src/api/blockchainEstimateFee.ts

```diff
@@ -82,6 +83,10 @@ export default class BlockchainEstimateFee extends AbstractMethod<'blockchainEst
         } else {
             const backend = await initBlockchain(coinInfo, this.postMessage, identity);
             feeInfo.levels = await backend.estimateFee(request || {});
+            // priority fees are not supported on Ethereum classic
```

Comment:
just prepare it in levels and handle unavailable data in Suite. We would need fallback to legacy anyways